### DISCUSSION TITLE
Add environment name in notifications

### DISCUSSION THREAD

**Discussion by ghost** [spatie/laravel-backup#1430] [Ideas]
May you please add the environment name to notifications, so we know what report is related to which? It's also needed for failed backup notifications and it's better to send the errors to logging channels instead of showing it in the notification itself.

**Comment by freekmurze** [comment]
Feel free to PR this.

**Comment by erikn69** [comment]
Maybe here 
https://github.com/spatie/laravel-backup/blob/1ee00b91e1862b488aa53cfe34f793b5b273b2dd/src/Notifications/BaseNotification.php#L19-L22
```php
public function applicationName(): string
{
    $appEnv = config('app.env');

    return (config('app.name') ?? config('app.url') ?? 'Laravel application') . ($appEnv ? '/'.$appEnv : '');
}
```
>It's also needed for failed backup notifications and it's better to send the errors to logging channels instead of showing it in the notification itself.

You can change that in config
